### PR TITLE
add common heading component and sidebar navigation

### PR DESCRIPTION
**Pull request by ayu-ayushisood**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -10,6 +10,9 @@
     position: absolute;
     font-family: 'Maven-pro', Arial, sans-serif;
 }
+a:hover {
```
Comment (karanasthana) [inline src/App.css]: Why would you want to do this ?

You will want to highlight tags on hover, right ?

If you're using <a> for other purposes, I would suuggest you to either use a class for it (Always). Or create a wrapper component on top of a and use that

File: src/Screens/experience/index.js [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
+import './style.css';
+import Sidebar from '../../components/Sidebar'
+import Heading from '../../components/Header'
 
 const Experience = () => {
     return (
-        <div >
-            In Experience
+        <div className="experience">
+           <Sidebar />
```
Comment (karanasthana) [inline src/Screens/experience/index.js:9]: How are you handling the width of the Sidebar ?

File: src/components/Sidebar/style.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    display: flex;
+    /* justify-content: space-between; */
+}
+
+.social-icons {
+    display: flex;
+    justify-content: space-between;
+}
+
+
```
Comment (karanasthana) [inline src/components/Sidebar/style.css]: extra lines, you can remove

File: src/components/Sidebar/index.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+                            <NavLink activeClassName="selected" to="/work">My Work</NavLink>
+                        </Nav.Link>
+                    </Nav.Item>
+                    <Nav.Item>
+                        <Nav.Link>Download Resume</Nav.Link>
+                    </Nav.Item>
+                </Nav>
+            </Navbar>
+            <div className="social-icons">
+                <LogoGithub
```
Comment (karanasthana) [inline src/components/Sidebar/index.js:45]: Only logos?

Are they WIP ? Or will you add links?

File: src/components/Sidebar/index.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+                    width="25px"
+                />
+                <div className="email">sood.ayushi30@gmail.com</div>
+            </div>
+            <div className="contact-container">
+                <Call
+                    height="25px"
+                    width="25px"
+                />
+                <div className="mobile">+91-9466460493</div>
```
Comment (karanasthana) [inline src/components/Sidebar/index.js]: you'll make your number PUBLIC ?

File: src/components/Sidebar/index.js (hunk 4 above)
Comment (ayu-ayushisood) [reply src/components/Sidebar/index.js:45]: wip, just got a structure ready

File: src/components/Sidebar/style.css (hunk 3 above)
Comment (ayu-ayushisood) [reply src/components/Sidebar/style.css]: ok

File: src/App.css (hunk 1 above)
Comment (ayu-ayushisood) [reply src/App.css]: done

File: src/components/Sidebar/index.js (hunk 5 above)
Comment (ayu-ayushisood) [reply src/components/Sidebar/index.js]: yeah. decided that phone number is not needed. Hence, removed.

**Review by karanasthana — APPROVED** [review]

**Pull request merged**